### DISCUSSION TITLE
"Experimental" Status of Popp ZB-Stick

### DISCUSSION THREAD

**Discussion by scoobydrvr** [Koenkk/zigbee2mqtt#14027] [Devices]
What does "Experimental" exactly mean in the supported devices list for Z2M?

This is coming from a fight I've been having trying to get a pair of IKEA E1812 shortcut buttons to work. I'm able to add them to the front end reliably but they continue to fail the Interview step. I've added myself to another thread for that particular issue but I'm beginning to wonder if my choice of adapter is the root cause of this issue.

**Comment by scoobydrvr** [comment]
Just to clean this up and consider it solved - I replaced this adapter with Tube's CC2652P2 based Zigbee to Ethernet/USB Serial Coordinator and it's operating without issue. The buttons above are connected and functional.

**Comment by Hedda** [comment]
> What does "Experimental" exactly mean in the supported devices list for Z2M?

FYI, there is more information and discussion about that if you follow these links, however, support for Silicon Labs EZSP adapters in Zigbee2MQTT is still relatively new and there is currently only one developer working on improving the support for the ezsp adapter in zigbee-herdsman (which is the library that Z2M depends on). 

https://github.com/Koenkk/zigbee2mqtt/discussions/13373

https://github.com/Koenkk/zigbee2mqtt/issues?q=is%3Aissue+is%3Aopen+%22EZSP+adapter%22

https://github.com/Koenkk/zigbee-herdsman/issues/319

https://github.com/Koenkk/zigbee-herdsman/pulls?q=is%3Apr+ezsp

Hopefully, stability will improve as more people in the Zigbee2MQTT community start to use it and help with development or as testers to actively contribute with proper and constructive bug reports.
